pyshtools/shio/icgem.py

```diff
@@ -0,0 +1,148 @@
+"""
+ICGEM-format read support
+"""
+
+from __future__ import absolute_import as _absolute_import
+from __future__ import division as _division
+
+import numpy as _np
+
+from pyshtools.shio.common import _derivative
+from pyshtools.utils.datetime import _yyyymmdd_to_year_fraction
+
+def read_icgem_gfc(filename, lmax=None, usecols=(3, 4), epoch=None):
+    """Read spherical harmonic coefficients from an ICGEM GFC ascii-formatted file.
+
+    This function only reads file with the gravity field spherical
+    harmonic coefficients.
+
+    Returns
+    -------
+    cilm : array
+        Array with the coefficients with the shape
+        (2, lmax + 1, lmax + 1) for the given epoch.
+    gm : float
+        Standart gravitational constant of the model, in m**3/s**2
+    r0 : float
+        Reference radius of the model, in meters.
+
+    Parameters
+    ----------
+    filename : str
+        The ascii-formatted filename containing the spherical harmonic coefficients.
+    usecols : sequence, optional
+        Which two columns to read. Can be used to read errors of the
+        coefficients instead of the coefficients themself. Can be (3, 4) or
+        (5, 6) or (7, 8). Default is (3, 4), i.e. read coefficients.
+    lmax : int, optional
+        Maximum degree to read from file, if None then maximum degree of the
```

Comment:
Maybe this is clearer? 

Maximum degree to read from the file. If lmax is None, less than 0, or greater than lmax_model, the maximum degree of the model will be used.